ecs-cli/modules/command/cluster_app.go

```diff
@@ -381,7 +384,20 @@ func validateCluster(clusterName string, ecsClient ecsclient.ECSClient) error {
 	if !isClusterActive {
 		return fmt.Errorf("Cluster '%s' is not active. Ensure that it exists", clusterName)
 	}
+	return nil
+}
 
+// deleteClusterPrompt prompts and checks for confirmation to delete the cluster
+func deleteClusterPrompt(reader *bufio.Reader) error {
+	fmt.Println("Are you sure you want to delete your cluster? [y/N]")
+	input, err := reader.ReadString('\n')
+	if err != nil {
+		return fmt.Errorf("Error reading input: %s", err.Error())
+	}
+	formattedInput := strings.ToLower(strings.TrimRight(input, "\n"))
```

Comment:
Can you change `TrimRight` to [`TrimSpace`](https://golang.org/pkg/strings/#TrimSpace)?  That should handle `\n` and any other white space that inadvertently gets added.